src/devices/cpu/m6809/m6809.h

```diff
@@ -270,6 +272,7 @@ class m6809_base_device : public cpu_device
 	bool is_ea_addressing_mode() { return m_addressing_mode == ADDRESSING_MODE_EA; }
 	uint16_t get_pending_interrupt();
 	void log_illegal();
+	devcb_write_line m_syncack_write_func;
```

Comment:
Please keep data members together.  Don’t add them in random places.

Reply:
Ok.